gapis/api/cmd_id.go

```diff
@@ -51,7 +51,9 @@ func (id CmdID) String() string {
 }
 
 // TODO(apbodnar) find a more appropriate place for this
-type CommandSubmissionKey struct {
-	SubmissionOrder uint64
-	CommandBuffer   int64
+type CmdSubmissionKey struct {
+	First  uint64
```

Comment:
Not really thrilled about this naming :)

but understand we don't want to leak Vulkanisms everywhere

Reply:
Can you make this
```
type CmdSubmissionKey []uint64
```

Then it is both generic, and does not need individualized named values.